styles/index.css

```diff
@@ -42,12 +47,118 @@ nav{
     box-shadow: rgba(50, 50, 93, 0.25) 0px 6px 12px -2px, rgba(0, 0, 0, 0.3) 0px 3px 7px -3px;
 }
 
-a:hover, i:hover{
-    color: var(--clr-primary);
+a, a:hover, i:hover{
+    color:var(--clr-primary);
+}
+
+.nav-pill-container{
+    display: inline;
+    width: 18rem;
+    display: flex;
+    justify-content: space-between;
 }
 
+.nav-pill{
+    display: inline;
+    list-style: none;
+    margin: 0 4px;
+}
+
+
 footer{
     bottom: 0;
     position: fixed;
     width: 100%;
+}
+
+.btn-img{
+    background-color:transparent;
+    border:none;
+}
+
+
+@media (max-width:460px){
+    .parent-container{
+        width: 90%;
+        overflow-x: hidden;
+        margin: 1rem;
+        padding: 0;
+    }
+
+    .header-container{
+        width: 95%;
+        margin: 1rem;
+        padding:1rem;
+        justify-content: space-between;
+    }
+    nav{
+        justify-content: flex-end;
+        align-items: flex-end;
+        width: 3rem;
+    }
+    .menu-btn{
+        display:inline;
+        background-color: transparent;
+        border: none;
+        font-size: 1rem;
+    }
+
+    .nav-pill-container{
+        top:4.5rem;
+        right: 4px;
+        position: absolute;
+        background-color: var(--clr-light-grey);
+        width: 9rem;
+        box-shadow: 0px 8px 16px 0px rgba(0,0,0,0.2);
+        z-index: 1;
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+        align-items: center;
+        display: none;
+
+    }
+    .nav-link {

```

Comment:
You can give a line space here too. As you have given at other places.

Reply:
Sure will do.